Add customization for God mode's minor mode lighter (#151)
emacsorphanage/god-mode: .github/README.md

## Visual indicators for God mode

God mode allows you to customize its minor mode lighter by customizing the `god-mode-lighter-string` variable and the `god-mode-lighter` face.
For example, if you don't want any lighter, you can set the string to nil.
On the other hand, if you want the lighter to stand out, you can change the face, e.g. by making it inherit from `error`.

> You can do the latter via `M-x customize-face RET god-mode-ligher` or via Lisp:

Reviewer: Typo and rephrasing:
```suggestion
You can do this via `M-x customize-face RET god-mode-lighter` or via Lisp:
```
